samples/snippets/remote_function_test.py

```diff
@@ -12,13 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import google.api_core.exceptions
+import google.cloud.bigquery_connection_v1
 import pytest
 
 import bigframes.pandas
 
 from . import remote_function
 
 
+# TODO(tswast): Once the connections are cleaned up in the sample test project
+# and https://github.com/GoogleCloudPlatform/python-docs-samples/issues/11720
+# is closed, we shouldn't need this because AFAIK we only use one BQ connection
+# in this sample.
+@pytest.fixture(autouse=True)
+def cleanup_connections() -> None:
+    client = google.cloud.bigquery_connection_v1.ConnectionServiceClient()
+
+    for conn in client.list_connections(
+        parent="projects/python-docs-samples-tests/locations/us"
+    ):
+        try:
+            int(conn.name.split("/")[-1].split("-")[0], base=16)
+        except ValueError:
+            print(f"Couldn't parse {conn.name}")
+            continue
+
+        try:
+            print(f"removing {conn.name}")
+            client.delete_connection(
+                google.cloud.bigquery_connection_v1.DeleteConnectionRequest(
+                    {"name": conn.name},
+                )
+            )
+        except google.api_core.exceptions.ResourceExhausted:
```

Comment:
NotFound can happen too if multiple samples sessions are running in parallel. Let's just stop the cleanup in case of any API exception.

```suggestion
        except google.api_core.exceptions.GoogleAPIError:
```